src/embedded.ts

```diff
@@ -259,6 +260,10 @@ export class RoomWidgetClient extends MatrixClient {
         if (sendContentLoaded) widgetApi.sendContentLoaded();
     }
 
+    public async supportUpdateState(): Promise<boolean> {
+        return (await this.widgetApi.getClientVersions())?.includes(UnstableApiVersion.MSC2762_UPDATE_STATE);
```

Comment:
`this.widgetApi.getClientVersions()` returns `Promise<ApiVersion[]>` so there is always an array. 

```suggestion
        return (await this.widgetApi.getClientVersions()).includes(UnstableApiVersion.MSC2762_UPDATE_STATE);
```

If this an error of typing in the widget api, the typing is wrong:

```suggestion
        return (await this.widgetApi.getClientVersions())?.includes(UnstableApiVersion.MSC2762_UPDATE_STATE) || false;
```